src/portal_visualization/builders/anndata_builders.py

```diff
@@ -77,7 +76,7 @@ def get_conf_cells(self, marker=None):
         ))
 
         vc = self._setup_anndata_view_config(vc, dataset, marker)
-        return get_conf_cells(vc)
+        return [vc]
```

Comment:
- See above: This method is now only responsible for returning confs
- Big change: Every class now returns a list of viewconfs. For most, it's only one element, but consistency in return types is a good thing.